packages/applications/ssr/src/components/pages/garanties-financières/mainLevée/annuler/AnnulerMainLevéeGarantiesFinancières.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+
+import Button from '@codegouvfr/react-dsfr/Button';
+import { useRouter } from 'next/navigation';
+import { useState } from 'react';
+
+import { Routes } from '@potentiel-applications/routes';
+
+import { ModalWithForm } from '@/components/molecules/ModalWithForm';
+
+import { annulerMainLevéeGarantiesFinancièresAction } from './annulerMainLevéeGarantiesFinancières.action';
+
+type AnnulerMainLevéeGarantiesFinancièresFormProps = {
+  identifiantProjet: string;
+};
+
+export const AnnulerMainLevéeGarantiesFinancières = ({
+  identifiantProjet,
+}: AnnulerMainLevéeGarantiesFinancièresFormProps) => {
+  const router = useRouter();
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  return (
+    <>
+      <Button priority="primary" onClick={() => setIsOpen(true)}>
+        Annuler sa demande de main-levée des garanties financières
```

Comment:
```suggestion
        Annuler la demande de main-levée des garanties financières
```

c'est plus une tournure qu'on utilise dans le projet